### PR TITLE
License

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

 * Update the README with list of dependencies
 * Remove dependencies in insight

# Testing this PR

Check that the build is green

cc @chris-allan @jrswedlow

### CODE REVIEW

**Comment by jburel** [comment]
--exclude

File: lib/licenses/README [hunk 1]
```diff
@@ -329,10 +577,54 @@
...
+  Project:      Minimal overhead Java logging
+  Web Site:     http://code.google.com/p/minlog/
+  License:      New BSD License
+  License File: bsd-new.txt
+
+
+  Artifacts:    native-lib-loader-*.jar
+  Project:      A library for loading native libraries.
+  Web Site:     http://github.com/scijava/native-lib-loader
+  License:      Free BSD License
```
Comment (mtbc) [inline lib/licenses/README]: usually "FreeBSD" single word

File: lib/licenses/README [hunk 2]
```diff
@@ -445,26 +776,66 @@
...
-  Web Site:     http://www.jgoodies.com/freeware/forms/index.html
-  License:      BSD
+  Artifacts:    xml-apis-*.jar
+  Project:      xml-commons provides an Apache-hosted set of DOM, SAX, and 
+                JAXP interfaces for use in other xml-based projects. Our hope is that we 
+                can standardize on both a common version and packaging scheme for these 
+                critical XML standards interfaces to make the lives of both our developers 
+                and users easier. The External Components portion of xml-commons contains 
+                interfaces that are defined by external standards organizations. For DOM, 
+                that's the W3C;
```
Comment (mtbc) [inline lib/licenses/README]: odd to end on a semicolon

**Comment by mtbc** [comment]
We seem to have a couple of libraries with no license noted. Have we at least a Trello card for investigating further?

**Comment by jburel** [comment]
@mtbc @will-moore is looking at the ones I left

**Comment by will-moore** [comment]
SpinBtn is long dead. No license info and the only url I can find is 404 http://www.softwareunity.com/jquery/JQuerySpinBtn/
Most jQuery stuff uses MIT license

**Comment by jburel** [comment]
I will add to  https://trello.com/c/zNVVhCR2/28-javascript-and-python-dependencies-review so we can look at removing it

File: lib/licenses/README [hunk 3]
```diff
@@ -564,162 +961,241 @@
...
+  License:      MIT License
+  License File: mit.txt
 
 
-  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/raphael/raphael-min.js
-  Project:      Raphael javascript library
-  Web Site:     http://raphaeljs.com/
-  License:      MIT license.
-  License File: jquery.txt
+  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/ie8_pngfix/*
```
Comment (will-moore) [inline lib/licenses/README]: I can't see that we use this anywhere.

File: lib/licenses/README [hunk 4]
```diff
@@ -564,162 +961,241 @@
...
+  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/jquery.quicksearch-1.0.js
+  Web Site:     https://github.com/riklomas/quicksearch
+  License:      MIT License
+  License File: mit.txt
+
+
+  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/jquery.selectboxes-2.2.6.js
+  Web Site:     
+  License:      MIT License or GNU GENERAL PUBLIC LICENSE
+  License File: mit.txt, gpl-v3.0.txt
```
Comment (will-moore) [inline lib/licenses/README:1242]: Can't see that we use this anywhere.

File: lib/licenses/README [hunk 5]
```diff
@@ -564,162 +961,241 @@
...
+  License:      MIT License
+  License File: mit.txt
 
 
-  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/d3*
-  Project:      d3.js
-  Web Site:     http://underscorejs.org/
-  License:      BSD
-  License File: bsd.txt
+  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/aops-1.3.js
```
Comment (will-moore) [inline lib/licenses/README:1203]: We don't use this aops-1.3.js library anywhere.

File: lib/licenses/README [hunk 6]
```diff
@@ -564,162 +961,241 @@
...
+
+  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/panojs-2.0.0/*
+  Project:      An interactive JavaScript widget for panning and zooming a panoramic image
+                stitched together dynamically from smaller tiles. 
+  Web Site:     http://www.dimin.net/software/panojs/
+  License:      MIT License
+  License File: mit.txt
+
+
+  Artifacts:    components/tools/OmeroWeb/omeroweb/webgateway/static/3rdparty/pie-1.0beta5/*
```
Comment (will-moore) [inline lib/licenses/README:1184]: I'm sure we can remove this now - only for old IE.

**Comment by jburel** [comment]
@will-moore I will not remove them from the code in the point release. 
I have added them to the card mentioned above.
They have similar licenses that some already listed

**Comment by sbesson** [comment]
@jburel: one thought while scanning `lib/licenses/README` is that we are not listing the Bio-Formats components (`formats-gpl`...). Is that a deliberate choice for our own dependencies?

**Comment by jburel** [comment]
I did not list any of our own dependencies.
Happy to do so if we think this is required

**Comment by chris-allan** [comment]
I would think it safe to link to the Bio-Formats dependencies rather than potentially introduce copy and paste errors and additional maintenance burden by also including them in `lib/licenses/README`.

**Comment by sbesson** [comment]
As we are independently maintaining the list Bio-Formats dependencies, I am definitely in favor of linking rather than trying to maintain in two places. I will review this list and flag all the low-level dependencies worth discussing.

File: lib/licenses/README [hunk 7]
```diff
@@ -658,6 +665,47 @@
...
+                memory efficiency improvements.
+                Used by Bio-Formats for OLE support (cxd, ipw, oib, zvi).
+                Used by VisBio overlays logic for XLS export feature.
+  Web Site:     http://jakarta.apache.org/poi/
+  License:      Apache License, Version 2.0
+  License File: apache-v2.0.txt
+
+
+  Artifacts:    ome-xml-*.jar
+  Project:      Java API to handle Microsoft MDB format (Access).
```
Comment (sbesson) [inline lib/licenses/README]: That's the same description as `ome-mdbtools`

**Review by sbesson — COMMENTED** [review]
Compared the content of the license README with the dependency report generated by Ivy and bundled under `share/{server,client}` in OMERO.server. 

The main missing dependencies I could find are:

- `formats-api`, `formats-gpl`, `formats-bsd`
- `bcprov-jdk14`, `bcmail-jdk14`
- `jts`
- `jaxrpc-api`
- `axis-saaj` should be `axis2-saaj`

**Comment by jburel** [comment]
@sbesson the list should now be complete

**Comment by sbesson** [comment]
The last commit addresses my comments from https://github.com/openmicroscopy/openmicroscopy/pull/5335#pullrequestreview-45710189

**Pull request merged**